docs.json

```
@@ -196,7 +196,8 @@
                   "hedera/tutorials/consensus",
                   "hedera/tutorials/consensus/submit-your-first-message",
                   "hedera/tutorials/consensus/submit-message-to-private-topic",
-                  "hedera/tutorials/consensus/query-messages-with-mirror-node"
+                  "hedera/tutorials/consensus/query-messages-with-mirror-node",
```

move `query messages with mirror node` tutorial to the correct position in the nav sidebar to match the tutorial order. unless it's arbitrary but i see you put it in 'next steps'

```suggestion
                  "hedera/tutorials/consensus/batch-anchor-verify-records-with-hcs",
                  "hedera/tutorials/consensus/query-messages-with-mirror-node"
```